src/renderer/entities/transaction/lib/extrinsicService.ts

```diff
@@ -64,20 +64,18 @@ export const getUnsignedTransaction: Record<
   [TransactionType.ASSET_TRANSFER]: (transaction, info, options, api) => {
     const palletName = transaction.args.palletName ?? 'assets';
     const methodArgs = api.tx[palletName].transfer.meta.args;
-    const rawArgs = [transaction.args.asset, transaction.args.target, transaction.args.amount];
+    const rawArgs = [transaction.args.asset, transaction.args.dest, transaction.args.value];
```

Comment:
It will fail on simple transaction case

Reply:
According to the following methods, everything is fine
@Asmadek maybe you could tell us what we're missing?

```javascript
type TransferParams = {
  chain: Chain;
  asset: Asset;
  accountId: AccountId;
  destination: string;
  amount: string;
  xcmData?: {
    args: {
      xcmFee: string;
      xcmAsset?: NonNullable<unknown>;
      xcmWeight: string;
      xcmDest?: NonNullable<unknown>;
      xcmBeneficiary?: NonNullable<unknown>;
      destinationChain: ChainId;
    };
    transactionType: TransactionType;
  };
};
function buildTransfer({ chain, accountId, destination, asset, amount, xcmData }: TransferParams): Transaction {
  let transactionType = asset.type ? TransferType[asset.type] : TransactionType.TRANSFER;
  if (xcmData) {
    transactionType = xcmData.transactionType;
  }

  const palletName =
    asset.typeExtras && 'palletName' in asset.typeExtras ? camelCase(asset.typeExtras.palletName) : 'assets';

  return {
    chainId: chain.chainId,
    address: toAddress(accountId, { prefix: chain.addressPrefix }),
    type: transactionType,
    args: {
      palletName,
      dest: toAddress(destination || TEST_ACCOUNTS[0], { prefix: chain.addressPrefix }),
      value: formatAmount(amount, asset.precision) || '1',
      ...(Boolean(asset.type) && { asset: getAssetId(asset) }),
      ...xcmData?.args,
    },
  };
}
```

Reply:
@johnthecat what's wrong with it? I didn't see any problems with transaction